pallets/loans/src/lib.rs

```diff
@@ -1046,7 +1046,7 @@ pub mod pallet {
 			pool_id: T::PoolId,
 		) -> Result<(T::Balance, u32), DispatchError> {
 			let rates = T::InterestAccrual::rates();
-			let prices = T::PriceRegistry::collection(&pool_id)?;
+			let prices = T::PriceRegistry::collection(&pool_id).unwrap_or_default();
```

Comment:
@mustermeiszer just doing this, we force to have an outdated collection to be empty, and later (comment below)...

Reply:
But this enables closing an epoch with outdated oracles, which is not wanted.

Reply:
why not use the `prices` as an parameter to `fn update_portfolio_valuation_for_pool `?

Reply:
Yes, that's the idea of allowing fake Oracle values.

Reply:
In this change, I just avoid errors. But the idea (maybe in another PR) is to modify the PoolNAV trait passing prices as a parameter